ports/esp32/machine_timer.c

```diff
@@ -108,7 +108,11 @@ static mp_obj_t machine_timer_make_new(const mp_obj_type_t *type, size_t n_args,
     mp_arg_check_num(n_args, n_kw, 1, MP_OBJ_FUN_ARGS_MAX, true);
 
     // Create the new timer.
-    machine_timer_obj_t *self = machine_timer_create(mp_obj_get_int(args[0]));
+    uint32_t timer_number = mp_obj_get_int(args[0]);
+    if (timer_number >= SOC_TIMER_GROUP_TOTAL_TIMERS) {
+        mp_raise_ValueError(MP_ERROR_TEXT("Invalid Timer number"));
```

Comment:
Please start error messages with lowercase (this is for consistency, and also improves the error message compression).

Reply:
Done & updated.

Reply:
Thanks!